File: packages/insomnia-httpsnippet/test/fixtures/output/ruby/native/application-form-encoded.rb
require 'uri'
require 'net/http'

url = URI("http://mockbin.com/har")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Post.new(url)
request["content-type"] = 'application/x-www-form-urlencoded'
request.body = "foo=bar&hello=world"

response = http.request(request)
puts response.read_body

File: packages/insomnia-httpsnippet/test/fixtures/output/go/native/application-form-encoded.go
package main

import (
	"fmt"
	"strings"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "http://mockbin.com/har"

	payload := strings.NewReader("foo=bar&hello=world")

	req, _ := http.NewRequest("POST", url, payload)

	req.Header.Add("content-type", "application/x-www-form-urlencoded")

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}
